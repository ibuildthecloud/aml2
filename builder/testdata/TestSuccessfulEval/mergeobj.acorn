a: arr: [1,2]
a: arr: [1,3]
a: b: 1
a: c: 2
a: b: 3
