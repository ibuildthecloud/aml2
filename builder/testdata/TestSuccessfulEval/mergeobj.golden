`{
  "a": {
    "arr": [
      1,
      3
    ],
    "b": 3,
    "c": 2
  }
}`
